todoman/cli.py

```diff
@@ -48,6 +48,24 @@ def _validate_date_param(ctx, param, val):
         raise click.BadParameter(e)
 
 
+def _validate_start_date_param(ctx, param, val):
+    if val is None:
+        return val
+    if val.startswith('before'):
```

Comment:
I left a comment [here](https://github.com/pimutils/todoman/pull/121#discussion_r102932267) as to why the tests fail